fastify.d.ts

```diff
@@ -69,6 +69,7 @@ declare namespace fastify {
     send: (payload?: any) => FastifyReply<HttpResponse>
     sent: boolean
     res: HttpResponse
+    context: any
```

Comment:
this should be something more specific. Moreover, it does not define `config`.

Reply:
Updated